src/main/java/core/selenium/webdrivers/BrowserFactory.java

```diff
@@ -28,6 +28,12 @@
         put(BrowserType.FIREFOX, new FireFox());
     }};
 
+    /**
+     * Contructor.
```

Comment:
In the description of the class is the constructor.... of what? please write clear

Reply:
I think is not necessary comment that because the constructor is by default and it is empty.